jira/client.py

```diff
@@ -5374,11 +5376,13 @@ def update_sprint(
             endDate (Optional[Any]): The start date for the sprint
             state: (Optional[str]): The state of the sprint
             goal: (Optional[str]): The goal of the sprint
+            autoStartStop: (Optional[bool]): Start and complete sprint automatically
+            incompleteIssuesDestinationId: (Optional[int]): After sprint completes, move open issues to this sprint id, -1 for backlog
 
         Returns:
             Dict[str, Any]
         """
-        payload = {}
+        payload : dict[str, Any] = {}
```

Comment:
mypy autoinferms dict[str, str], and it would not be compatible with bool nor int